packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```diff
@@ -708,3 +732,8 @@ popd
 %{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.cfg
 %{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.env
 %{_cross_unitdir}/nvidia-fabricmanager.service
+
+%files imex
+%license imex-%{fm_arch}-%{tesla_ver}-archive/usr/share/licenses/nvidia-imex-%{tesla_major}/LICENSE
```

Comment:
Is the license different from the overall package license, such that the fm and imex subpackages should have a different license statement?